packages/kbn-std/src/is_internal_url.test.ts

```diff
@@ -61,6 +61,23 @@ describe('isInternalURL', () => {
       const href = `${basePath}/app/kibana/../../../management`;
       expect(isInternalURL(href, basePath)).toBe(false);
     });
+
+    it('should return `false` if absolute URL contains tabs or new lines', () => {
+      for (const char of ['\t', '\n', '\r', '\t\n\r']) {
+        for (const url of [
+          `/${char}/${basePath}app/kibana`,
+          `/${char}//${basePath}app/kibana`,
+          `//${char}/${basePath}app/kibana`,
+          `htt${char}ps://example.com${basePath}`,
+          `htt${char}p://example.org:5601${basePath}`,
+          `file${char}://example.com${basePath}`,
+          `htt${char}p://example.org:5601${char}${basePath}`,
+          `https:/${char}/example.com${char}${basePath}`,
+        ]) {
```

Comment:
Wouldn't these all return false anyway, even without the special character? Should we have at least one test case that would otherwise return true, if not for the tab/newline?

Reply:
I've explained the idea in https://github.com/elastic/kibana/pull/183521/files#r1603774927, but in this specific case (with the base path) you're right - these all seem to return `false` even with the previous code, which is not what we want. Let me take a look.